\documentclass[11pt]{article}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage[backend=bibtex, style=authoryear]{biblatex}

\usepackage[mmddyyyy,hhmmss]{datetime}
\usepackage{xcolor}

\definecolor{nicholasCol}{RGB}{203,97,63}
\newcommand{\nicholas}[1]{{\color{nicholasCol} [\textbf{NS:} #1 (\today\ \currenttime)]}}

% \addbibresource{~/Dropbox/TAGreview/PhDreferences.bib}

\textwidth=470pt
\oddsidemargin=0pt
\topmargin=0pt
\headheight=0pt
\textheight=650pt
\headsep=0pt

\pdfpagewidth=\paperwidth
\pdfpageheight=\paperheight

\title{Statement of Research}


\author{Nicholas Santantonio}
\date{\today}


\newcommand{\gxe}{G$\times$E}
\newcommand{\gxg}{G$\times$G}


\begin{document}

\section*{\centering Statement of Research}
\begin{center} Nicholas Santantonio \end{center}

% \subsection*{Introduction}

% gxg cows
% cover crop to sequester carbon, 
% htp, 
% weather data
% crop modeling,
% abiotic biotic adaptation
% climate change
% biofuel
% sustainable cropping systems. 


% digital ag
% connect with engineering dept. 
% Fed capacity Hatch funds used only for breeding  

% Lead by Professors Royse Murphy and Don Viands, the Cornell forage breeding program has a rich legacy of providing forage crop varieties to farmers in the Northeast US, as well as collaboration nation-wide, a legacy that I hope to contribute to. 

% Forage research has been drastically under-funded compared to food crops, but renewed interest in sustainable farming has brought them back into the limelight. 

\noindent In the face of climate change, farms will have less access to agronomic inputs while they strive to update and maintain sustainable agricultural practices. This leaves genetics as the primary target for improvement of our food systems. Interest in sustainable agriculture has brought renewed attention to forage and industrial crops, which can increase soil nitrogen, sequester carbon and reduce weed populations. These crops can also be adapted to marginal lands, leaving more flexibility for farmers to use and less  competition with food crops. Implementation of $21^{st}$ century breeding technologies has lagged in forages due to the complexity of the genetics, phenotyping and costs associated with evaluation of quality.



\subsection*{Short-term gains for alfalfa}

Alfalfa, known as the ``Queen of forages'' for its high protein content, was valued at 127 million for production in NY 2018 (USDA 2019). Introgression of beneficial alleles in alfalfa is exceedingly difficult \nicholas{arduous?} due to high inbreeding depression and the autotetraploid nature of the crop. As an obligate out-crosser, alfalfa must always be bred on a population level, where varieties are released as synthetics to avoid inbreeding and take advantage of population level heterosis. This has limited implementation of marker-based selection because large numbers of individuals must be genotyped and inter-mated to avoid inbreeding in future generations. Further complicating the use of markers is which individual(s) should be genotyped to represent a given variety. \nicholas{remove this sentence?} Generally, single individuals are not representative of the variety as a whole, and genotyping large numbers of individuals from each variety is costly and restrictive.

To implement the use of genome-wide information in alfalfa, I propose a new population-level genomic selection framework (popGS). Tissue from many individuals within a variety or breeding population will be bulked for DNA extraction and sequence-based genotyping. This will reduce genotyping costs while still allowing the standard genomic prediction framework to operate on allele counts in the population as opposed to allele counts within a single individual. PopGS will allow for prediction of additive effects for genetic gain, as well as dominance effects to exploit population level heterosis. This genotyping method will also be used to track allele frequency changes through time. Loci under natural selection for stand persistence will be identified and appropriately weighted in the popGS model.

A training set will be developed by genotyping varieties and breeding material from the Cornell forage breeding program that have been recently phenotyped. Moving forward, all new entries into the program will be genotyped in this fashion. New populations will be formed using mathematical optimization to determine optimal contributions of parent varieties. Populations will then be planted in optimal proportions and randomly inter-mated to maximize beneficial population allele frequencies in the resulting seed lot while minimizing inbreeding. Once shown to be effective, this strategy could revolutionize the way alfalfa seed is produced. Instead of needing several generations of inter-mating within a new synthetic to produce sufficient commercial seed, new varieties would be defined by their \emph{parents}. Whole fields could be planted in optimal parental proportions to produce farmer seed directly, thus making new varieties almost instantaneously available while maximizing the benefit of population level heterosis.
% allows popGS to work in a fluid fashion, where new varieties are constructed by mixing the best alleles together in optimal frequencies to leverage high additive and non-additive effects in the resulting populations. %This methodology may also be used for integration of CRISPR gene edits from single individuals into elite germplasm at high frequencies.

% CRISPR methodology is still in its infancy, but once individual plants can be transformed at all four alleles, crossing into elite germplasm will still be a challenge. Half-sib families will be at best duplex for the CRISPR allele, with further reduction in frequencies due to the inability to intracross or backcross. PopGS can be used to enrich CRISPR allele frequencies until phenotypic evaluation is feasible.

% A modified rhAmpSeq approach will be built to capture sequences likely to differ at more than one site such that multiple alleles can be tracked.  One approach may include building primer sets that are anchored in exons, but span introns, in order to maximize the likelihood of amplification and sequence differences. Specific motifs? THIS PARAGRAPH NEEDS WORK!

In pursuit of this goal, I conceived of, wrote and was awarded a US Alfalfa Farmer Research Initiative (USAFRI) grant with Kelly Robbins, Don Viands and Julie Hansen as PI and co-PIs, respectively. Funds were procured to re-sequence eight alfalfa varieties currently under phenotypic evaluation in Geneva NY, as well as the nine historic germplasm sources of alfalfa (Segovia-Lerma et al. 2003, Jour. vol. no.). \nicholas{necessary?} A diallel population formed from the historic germplasm sources was previously developed and phenotyped in Las Cruces, NM (Segovia-Lerma 2004). These two populations will be used to augment theoretical results with empirical evidence, as well as help develop an affordable sequence-based genotyping platform, and a potential collaboration with Breeding Insight. The proposal is available on request. 


\subsection*{Developing a Forage Phenome}

Forages are typically harvested multiple times per year and are often perennial, requiring several years of evaluation. Forage quality must be measured to gauge animal nutrition and digestibility, but is expensive to phenotype. This makes for a heavy phenotypic burden on the breeding program that stands to benefit from the aid of Digital Ag, with opportunities for collaborations with the Cornell Initiative for Digital Agriculture.

Forage cover crops are highly effective for soil rejuvenation, but the genetic variability of these processes is largely unknown because looking underground is notoriously difficult. Root phenotypes will be key to modeling carbon and nitrogen cycling, drought response, and nodulation. Tap- and fine-root architecture and turnover phenotypes can be collected through time by sampling 1 meter cores within plots, but strategies to economically select for these traits are desperately needed. %Tap root morphology will be measured using cores within the drill strip row, while fine root structure will be evaluated from cores sampled between drill strips within each plot. Root phenotypes will be taken twice per year, after the first and last harvests of the season, to determine root growth cycles. 

To mitigate the phenotypic burden of multiple years, cuts and the measuring of root traits, multi-spectral imaging data will be collected regularly throughout each growth cycle to produce high-throughput phenotypes (HTP). Genetic correlations of low-throughput phenotypes (LTPs) and HTPs will be estimated and used to build genomic prediction models to predict unobserved LTPs. 

% Alfalfa cuts as lactation curves on multiple pregnancies

HTPs will also be used to develop population specific growth curves to model development above and below ground under differing biotic and abiotic stressors. Instead of waiting for three year trials to come to completion, rapid-cycle selection will be performed on a yearly basis using popGS and all available HTPs and LTPs from historical and in-field material. Models will be updated annually with HTP and LTP information from plots in the field. 

As part of the USAFRI grant, I obtained an FAA remote pilot certification and collected semi-weekly, multi-spectral images of an alfalfa variety trial currently under evaluation in Geneva NY. These results will be used as supporting evidence for a future grant proposal to the USDA (NIFA?) to evaluate incorporation of HTP into a rapid-cycle forage breeding program. The efficacy of an HTP phenotyping strategy will be assessed by prediction of LTPs using genome wide information and HTPs. Moving forward I intend to expand the size and number of field trials, using HTPs to limit the phenotypic burden of more extensive field trials. 

% selection will be assessed using several selection schemes, where crosses will be made based on genotype predictions alone, genotype plus HTP, and genotype plus HTP and LTP, along with selection on LTPs alone as a control.   
% (i.e. popGS) of cycle 0 at year 1 (C0Y1), genotype predictions including HTP at year 1, and phenotypic information from year 1 alone. After 1 round of selection, the resulting populations will be planted in the field for evaluation (C1.1). Additional phenotypes from cycle 0, year 2 (C0Y2) and cycle 1, year 1 (C1Y1) will be used to update prediction models for an additional round of selection from both C0 and C1 to produce C1.2 and C2.1 populations, respectively. Three year evaluation of these three generations will be contrasted to determine the merit of HTP assisted popGS. 

\subsection*{Long-term sustainability}

Forage crops are only one part of an agronomic biological system which includes soil microorganisms, the animals that feed on the forages and the microbiome of those animals. Other than host-pathogen interactions, little attention has been paid to genomic interactions between these organisms (i.e. \gxg), despite an overall notion that they are important. These interactions can be thought of as a special case of the \gxe\ problem, where the covariance of the ``environment'' (e.g. soil microorganisms) can be determined by genotyping that ``environment''. Two interacting systems will be targeted for research into \gxg: the interaction between forage and soil microbes, and the interaction between animal and feed. 

% Interactions can then be modeled using standard \gxe\ machinery, where the genetic covariance of the biological combinations is a Kronecker product of their genetic relationships. 

As a legume, alfalfa and other leguminous forages can form symbiotic relationships with nitrogen fixing bacteria, \emph{Rhizobium}. Unfortunately, the signaling and infection process for nodulation is typically reduced or absent under moderate to high soil nitrogen levels. Genetic increases in nodulation could allow for the use of less chemical fertilizer, reducing the environmental impact of nitrogen runoff. The use of \gxg\ prediction models will allow for simultaneous selection of host variety and symbiont. Whole genome expression of host and symbiont as an intermediate interaction phenotype may also aid in understanding the genetic variation in signaling between the two species through time, under contrasting levels of available nitrogen. This may help us to understand how to select for changes in gene expression to increase the rate of colonization, as well as nitrogenase and leghemaglobin expression, even when some nitrogen is available in the soil. 

% targeted selection of host-symbiont combinations for testing that are likely to be beneficial, and 
% Collection, mutagenesis, and transformation of \emph{Rhizobium} may enable the genetic variation necessary 

% On the other side of the plant, 

There has been considerable research into nutrition of different feeds and how they interact with the microbiome of the rumen in dairy cows. While measures of forage quality are considered, little is known about the effect of different varieties on the animal and its microbiome. I seek to establish a collaboration with the animal nutrition department, local dairy farmers, and animal genetics companies that serve the Northeast to investigate the potential for synergistic forage and animal breeding. Instead of breeding animals independently of their feed, we can start to breed specialized animals to specialized feeds. While it will take time to establish relationships with animal breeders, dairy and hay farmers, bringing these communities together would set precedent for future long term integrated breeding operations.


\subsection*{Research Philosophy}

In the era of big data, the number of testable hypotheses is seemingly limitless. A shift away from small designed experiments to large observational studies at the breeding program or whole organism scale is inevitable. A traditional breeding program generates a plethora of phenotypic data that is used to make yearly breeding decisions, and subsequently discarded. If genotyped, these materials become treasure troves of data for asking questions, as well as making breeding decisions. This does not mean that we should cease the design and execution of experiments to address specific hypotheses, but we cannot ignore the valuable resource of observational data being collected, typically at great expense. Genotyping at this scale is feasible given the drastic reduction in costs and availability of third party services, and can be offset by clever experimental design that trades replication at an individual level for replication at the genetic level.

I believe in the collaborative model, where breeding programs do not operate in isolation. They share germplasm, resources, expertise, and most importantly, ideas. Unlike germplasm, ideas also have the merit of being species flexible. I intend to build a collaborative effort at Virginia Tech to aid all breeding programs to build foundational capabilities to increase efficiency of varietal development. As climate change progresses, Virginia will likely experience rising temperatures and more severe weather events, making crop production more difficult. Heat, drought, intense storms and hard frosts will be the new norm, and we must work together to do our part in defending our food security through accelerated genetic improvement. 


% \section{other ideas}

% There is also interest in short term alfalfa crops, which can provide a large percentage of the field health gain of a full alfalfa cycle in only one or two years. This well have to be coupled with selection for high seed production under low planting density, a known a bottle neck in an alfalfa agronomic system. %There is also interest in short term alfalfa crops, which can provide a large percentage of the field health gain of a full alfalfa cycle in only one or two years. 

% \subsection*{Personnel}

% I would like to hire one post doctoral associate and one graduate student for the first year, with the potential for additional graduate students and post docs for the following year depending on grant funding. The post doc will work with me to write the popGS theory and simulation paper. The graduate student will be responsible for development of the population level genotyping platform using seed from the previously phenotyped population we select for the POC, with guidance from myself and the post doc. This will lead to two companion manuscripts, with the first on the genotyping platform, and the second on the results from the popGS POC study. 

% The forage breeding program at Cornell consists of a strong team, whose knowledge and experience will be invaluable to implementing a successful GS breeding program. Additional personnel will be brought on as necessary, including a technician to operate and manage all HTP equipment and activities.

% \subsection*{A forage community}

% One of the challenges is the lack of large phenotypic records for alfalfa in any alfalfa breeding program.

% To encourage collaboration, data sharing and a global initiative, I would like to work toward an initiative to create a North American forage database using an available platform such as T3, or BassavaBase. All tools built by our lab will be made publicly available through the (BRAPI/ galaxy etc?) to allow forage breeding programs throughout the world to make more informed mating decisions. Other programs that produce useful tools for forage breeding will also be encouraged to link them to the $\alpha\alpha$Base through BRAPI.

% % Th to encourage public (and perhaps private?) breeding programs to contribute and use data to . We want to implement a genomic prediction tool for alfalfa breeders that will also design idealized mating schemes .

% I intend to continue to foster collaborations built by Dr. Don Viands and his forage breeding research team, while hoping to expand them with new research groups, such as that of Animal Nutrition (Cornell), Ian Ray (NMSU), Charlie Brummer (UC Davis), Maria Monteros (Noble Foundation), and Debby Samac (USDA ARS at UMN, and the new forage breeder they are currently looking to hire). The forage breeding program at Cornell consists of a strong team, whose knowledge and experience will be invaluable to implementing a successful GS breeding program. Additional personnel will be brought on as necessary, including a technician to operate and manage all HTP equipment and activities.



% If proven useful, I intend to establish collaborations to implement the strategies outlined here in other forage programs in the US and abroad.    

% Finally, I recognize that the forage breeding project currently focuses on many forages other than alfalfa. It is my intent to keep those projects, while incorporating new technologies for genetic gain in these crop. My focus on alfalfa is driven by the need for reliable GS methods in a polyploid crop that must be bred at the population level. I am open to new species as well, especially if a graduate student has a desire to investigate such species.  



% The third grant proposal will be to unify breeding efforts. Genotype by genotype interactions. 

% Field establishment 


% Other long term goals include:

% Collaborate with the Agro-ecology biochemistry.  CRISPR will be used to change both the Rhizobium as well as the plant. 

% Collaboration with the animal nutrition department. sciences


% A technician will be recruited to operate the HTP platform. 

% Implementation of marker-based selection has also lagged because large numbers of individuals must be genotyped to avoid inbreeding depression in future generations. 






\end{document}
